.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!--
+Have you read our official documentation? This includes a FAQ section which may contain info that can solve your problem.
+You can find it here: https://tediousjs.github.io/tedious/
+-->
+
+**Software versions**
+* Tedious:
+* SQL Server:
+* Node.js:
+
+**Additional Libraries Used and Versions**
+Provide any additional libraries used on top of Tedious (e.g. node-mssql v8.0.2).
+
+**Table schema**
+Provide the table schema to reproduce the issue.
+
+**Problem description**
+Provide full details of the problem.
```

Comment:
I assume that we want users to put in their repo steps here? Maybe we can also add a notice for users to remember to provide their connection configuration settings. This could be useful for us to detect some basic issues like encrypt = false kind of problem.

Reply:
Will add a config section and ask users to add repro steps